sql/src/test/java/io/crate/analyze/ExplainAnalyzerTest.java

```diff
@@ -47,8 +48,17 @@ public void testExplain() throws Exception {
         ExplainAnalyzedStatement stmt = e.analyze("explain select id from sys.cluster");
         assertNotNull(stmt.statement());
         assertThat(stmt.statement(), instanceOf(QueriedRelation.class));
+        assertThat(stmt.analyze(), is(false));
     }
 
+    @Test
+    public void testExplainAnalyze() {
```

Comment:
maybe rename to `testAnalyzePropertyIsSetOnExplainAnalyze` and remove the other assertions. Or rename to something like `testAllPropertiesAreSensibleOnExplainAnalyze`. We regularly have difficulties to figure out what a test was supposed to test if we've to touch it due to a refactoring.